allennlp/modules/token_embedders/elmo_token_embedder.py

```diff
@@ -0,0 +1,219 @@
+'''
+Compute the context insensitive token embeddings from pretrained biLMs.
+'''
+import json
+import logging
+
+import numpy
+import h5py
+
+from overrides import overrides
+
+import torch
+from torch.autograd import Variable
+
+from allennlp.common import Params
+from allennlp.common.checks import ConfigurationError
+from allennlp.common.file_utils import cached_path
+from allennlp.modules.token_embedders.token_embedder import TokenEmbedder
+from allennlp.modules.highway import Highway
+from allennlp.nn.util import add_bos_eos
+from allennlp.data.token_indexers.elmo_indexer import ELMoCharacterMapper
+from allennlp.data import Vocabulary
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+DTYPE = 'float32'
+
+# pylint: disable=invalid-name,protected-access,attribute-defined-outside-init,bad-continuation
```

Comment:
Can you fix these, or disable them where they are used? The invalid name, protected access and bad-continuation ones should be fix-able.

Reply:
done